examples/examples_test.go

```diff
@@ -26,11 +26,21 @@ func TestAccMinimal(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccDurableFunctions(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "durable-functions"),
+			RunUpdateTest: true
```

Comment:
You have to put `,` at the end of each line (see build errors). Sorry, I didn't realize that 2.0 thing was all over the place, feel free to revert that changes - they shouldn't be your problem.